features/legacy_snippets.feature

```diff
@@ -301,7 +301,7 @@ Feature: Legacy Snippets
       --- FeatureContext has missing steps. Define them with these snippets:
 
           #[Given('I have a package v2.5')]
-          public function iHaveAPackageV(): void
+          public function iHaveAPackageV25(): void
```

Comment:
It would be trivial to strip trailing numbers as before. However I'm not sure it's warranted (bearing in mind this number is in the hardcoded step text, it is not a placeholder).

Reply:
this case is fine to me